Modules/FindCUDA.cmake

```diff
@@ -439,6 +438,8 @@ endmacro()
 ###############################################################################
 ###############################################################################
 
+option(CUDA_USE_STATIC_CUDA_RUNTIME "Use the static version of the CUDA runtime library if available" ON)
```

Comment:
Well, I found that I could not possibly override it from outside no matter what. Even though it looked like it should have worked. This way override works, also I find it more intuitive for all options decraled upfront. Keeping the option enabled only if we find static lib is still done, further down.